packages/lesswrong/components/posts/PostsEditForm.tsx

```diff
@@ -47,7 +47,11 @@ const PostsEditForm = ({ documentId, eventForm, classes }: {
     collectionName: "Posts",
     fragmentName: 'SuggestAlignmentPost',
   })
-  
+
+  if (document?.fmCrosspost?.isCrosspost && !document?.fmCrosspost?.hostedHere) {
+    return <ForeignCrosspostEditForm post={document as PostsWithNavigation} />;
```

Comment:
What's `document` missing that we need the type cast here?

Reply:
Good catch - this is a TODO that I missed. I've just made some small changes so the `as` shouldn't be needed anymore.